packages/table/src/headers/columnHeader.tsx

```diff
@@ -168,50 +205,98 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
         });
         const cellLoading = cell.props.loading != null ? cell.props.loading : loading;
         const isColumnSelected = Regions.hasFullColumn(selectedRegions, columnIndex);
-        const cellProps: IColumnHeaderCellProps = { className, isColumnSelected, loading: cellLoading };
+        const isColumnTemporarilyReorderable = this.isColumnTemporarilyReorderable(isColumnSelected);
```

Comment:
does it matter that it's "temporary?" i think "currently" might be a better word, cuz it's determined by props and state. but that's just pure react, so perhaps we don't need such a word at all.

usage doesn't suggest any notion of temporary--you immediately rename it to `isColumnReorderable`, which as a prop is understood to only matter in the now, not next render.